features/create_membership_application.feature

```diff
@@ -23,6 +28,15 @@ Feature: As a visitor
     Then I should be on the landing page
     And I should see "Thank you, Your application has been submitted"
 
+  Scenario: Visitor can see which fields are required
+    Given I am on the landing page
+    And I click on "Apply for membership"
+    Then the required field "Company Name" should have class req
```

Comment:
should be `Then the required field "whatever" should have a required field indicator`
because 
- you want to separate out testing that _this field is required_, and _what the required field indicator_ is
- a _required field indicator_ may be more than one thing (usually it's both color and some additional visual thing -- like "*").

(Many other fields on the site are likely to need this ability: to be tagged as required.  So you want to be able to separate the two ideas so that they can be used independently.)

You can then have a separate scenario (or RSpec) that proves that your _required field indicator_ is exactly what you want it to be.